app/views/vendors/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1>Vendors Main Page</h1>
+
+<h2><%= link_to "Search for a Vendor Directly", search_vendors_path %></h2>
+
+<a href="/vendors/new">
+<div class="new">
+  <div class="vendor_back">
```

Comment:
CSS style guides state that multiple-word styles should be separated by `-` rather than `_`